File: bulk/csharp/SqlInjection02.cs
// CWE-89: SQL Injection via string interpolation
using System.Data.SqlClient;
public class SqlInjection02 {
    public object Search(SqlConnection conn, string term) {
        var cmd = new SqlCommand($"SELECT * FROM Products WHERE Name LIKE '%{term}%'", conn);
        return cmd.ExecuteReader();
    }
}

File: bulk/java/SqlInjection02.java
// CWE-89: SQL Injection in JPA native query
import javax.persistence.*;
public class SqlInjection02 {
    @PersistenceContext EntityManager em;
    public Object findByEmail(String email) {
        return em.createNativeQuery("SELECT * FROM users WHERE email = '" + email + "'").getSingleResult();
    }
}
